contracts/seed/Seed.sol

```diff
@@ -216,19 +223,30 @@ contract Seed {
     /**
       * @dev                     Claim locked tokens.
       * @param _locker           The address of the locker.
+      * @param _maxClaimAmount   The maximum amount of seed token a users wants to claim.
     */
-    function claimLock(address _locker) public allowedToClaim {
-        uint16 daysVested;
+    function claimLock(address _locker, uint256 _maxClaimAmount) public allowedToClaim {
```

Comment:
as discussed with Arseny, we are thinking of using msg.sender rather than taking it as a parameter. If we want to keep the ```_locker``` parameter, we might need to add some access control as we are not checking if the ```_locker == msg.sender```. Review required here.

Reply:
I always assumed that the intention here would be to allow anyone to claim on behalf of anyone else.  Is there any harm in allowing that?  There certainly are potential benefits.

Reply:
Okay, so should we keep it the same? And also should we have some sort of mapping so that we can know who can claim on behalf of someone else?
@dkent600

Reply:
I don't think there is any need for a mapping.  I would check with Luuk+Pat about whether we want folks to be able to claim on behalf of others.  I would vote Yes, but I don't know everything....

Reply:
Okay, so as of now I'll keep the ```_locker``` parameter.